xml/System/TimeSpan.xml

```diff
@@ -5108,8 +5108,8 @@ In some cases, when a time interval component in the string to be parsed contain
       <MemberSignature Language="VB.NET" Value="Public Shared Function TryParseExact (input As String, format As String, formatProvider As IFormatProvider, ByRef result As TimeSpan) As Boolean" />
       <MemberSignature Language="F#" Value="static member TryParseExact : string * string * IFormatProvider * TimeSpan -&gt; bool" Usage="System.TimeSpan.TryParseExact (input, format, formatProvider, result)" />
       <MemberSignature Language="C++ CLI" Value="public:&#xA; static bool TryParseExact(System::String ^ input, System::String ^ format, IFormatProvider ^ formatProvider, [Runtime::InteropServices::Out] TimeSpan % result);" />
-      <MemberSignature Language="C#" Value="public static bool TryParseExact (string? input, string? format, IFormatProvider? formatProvider, out TimeSpan? result);" FrameworkAlternate="net-5.0" />
-      <MemberSignature Language="C#" Value="public static bool TryParseExact (string? input, string format, IFormatProvider? formatProvider, out TimeSpan? result);" FrameworkAlternate="netcore-3.0;netcore-3.1" />
+      <MemberSignature Language="C#" Value="public static bool TryParseExact (string? input, string? format, IFormatProvider? formatProvider, out TimeSpan result);" FrameworkAlternate="net-5.0" />
```

Comment:
This looks correct: https://github.com/dotnet/runtime/blob/366cd23a82fc44ea2c613c36bd4bfdbf8c310e56/src/libraries/System.Private.CoreLib/src/System/TimeSpan.cs#L354